ouroboros-network/src/Ouroboros/Network/Socket.hs

```diff
@@ -49,6 +62,15 @@ import           Ouroboros.Network.Mux.Interface
 
 import           Text.Printf
 
+-- |
+-- We assume that a TCP segment size of 1440 bytes with initial window of size
+-- 4.  This sets upper limit of 5760 bytes on each message of handshake
+-- protocol.  If the limit is exceeded, then @'NotEnoughInput'@ exception will
+-- be thrown.
```

Comment:
I still don't understand this. Why is it not enough input? Isn't it too much input?

Reply:
The comment is stale, I'll update it.